### PR TITLE
Add contributing guidelines

### PR DESCRIPTION
**Pull request by Nek-**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+Contributing to Woketo
+======================
+
+Code style
+----------
+
+Woketo follows most parts of the code-style of [PSR](http://www.php-fig.org/psr/) & [Symfony](http://symfony.com/doc/current/contributing/code/standards.html) execept these parts:
```
Comment (valanz) [inline CONTRIBUTING.md]: "except for these points"

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+1. `return` clause must be explicit. That means that when you return null, you should precise `return null;`
+2. comments in docblock should be aligned
+3. We put spaces around the `.` of concatenation and the `=` of default parameter in functions
+
+*This code-style is a recommandation but can be pointed out in a pull-request code review.*
```
Comment (valanz) [inline CONTRIBUTING.md]: **recommendation**

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+Tests
+-----
+
+### Every patch or new behavior must have his unit test (phpunit), even if the functional test exists via autobahntestsuite.
```
Comment (folliked) [inline CONTRIBUTING.md]: AutobahnTestsuite

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+*This is a good-enough motivation to* ***not*** *merge a pull-request.*
+
+Documentation
+-------------
+
+Each time you can relate a method to a part of the [RFC](https://tools.ietf.org/html/rfc6455). Please add the most precise link to the concerned part of the RFC.
+
+Commits and squash
```
Comment (folliked) [inline CONTRIBUTING.md]: squashes ?

**Pull request merged**